templates/form_theme_admin.html.twig

```diff
@@ -0,0 +1,14 @@
+{% extends '@SonataAdmin/Form/form_admin_fields.html.twig' %}
+
+{% block app_admin_mooc_mooc_image_file_image_row -%}
+    {{ block('admin_image_with_url_row') }}
+{%- endblock app_admin_mooc_mooc_image_file_image_row %}
```

Comment:
can you remove the block name here pls, as the second endblock L14 is without the name?